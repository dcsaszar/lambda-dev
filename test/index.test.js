const CliTest = require('command-line-test');

describe('index', () => {
  it('Should print help text with no arguments', async () => {
    const cli = new CliTest();
    const { error, stdout, stderr } = await cli.exec('node .');

    expect(error).toMatchSnapshot();
    expect(stdout).toEqual(null);
    expect(stderr).toEqual(null);
  });
});
