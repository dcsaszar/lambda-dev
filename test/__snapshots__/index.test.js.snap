// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index Should print help text with no arguments 1`] = `
[Error: Command failed: node .
 <command>

Commands:
  serve <entry>           Lambda Development Server
  build <entry> <target>  Bundle Lambda functions

Options:
  --help      Show help                                                [boolean]
  --version   Show version number                                      [boolean]
  --node, -n  Target Node.js version, used with @babel/preset-env
                                                      [string] [default: "8.10"]

Not enough non-option arguments: got 0, need at least 1
]
`;
